### ISSUE TITLE
Failed to load the ionide-webview package

### ISSUE THREAD

**Issue by Yoshi325** [ionide/ionide-webview#4]
[Enter steps to reproduce below:]

1. Update to version 1.0.2 of ionide-webview
2. Start Atom

**Atom Version**: 1.1.0
**System**: Mac OS X 10.10.5
**Thrown From**: [ionide-webview](https://github.com/Ionide/ionide-webview) package, v1.0.2

### Stack Trace

Failed to load the ionide-webview package

```
At Unrecognised input in ~/.atom/packages/ionide-webview/styles/core.less:23:2

LessError: Unrecognised input
  at ~/.atom/packages/ionide-webview/styles/core.less:23:2
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "seti-ui",
      "visual-studio-dark-syntax"
    ],
    "projectHome": "~/Code/",
    "ignoredNames": [
      ".DS_Store",
      ".git",
      "*.pyc",
      "__pycache__",
      "__init__.py",
      "__main__.py"
    ],
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "atom-innerspace",
      "atom-sharp"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-jade, v0.3.0
atom-jinja2, v0.5.0
atom-python-debugger, v0.3.4
atom-yeoman, v0.3.15
build, v0.48.0
color-picker, v2.0.13
compare-files, v0.6.2
file-icons, v1.6.11
ionide-fake, v1.2.2
ionide-fsharp, v1.4.5
ionide-fsi, v1.0.9
ionide-installer, v1.3.0
ionide-paket, v2.2.2
ionide-webview, v1.0.2
ionide-yeoman, v1.0.2
json-schema, v0.1.15
language-batch, v0.3.3
language-diff, v0.3.1
language-fish-shell, v1.0.2
language-nginx, v0.6.1
language-powershell, v2.1.0
language-restructuredtext, v0.9.0
language-shade, v0.0.1
line-diff-details, v1.1.3
line-ending-converter, v1.3.2
linter, v1.10.0
linter-pylint, v1.0.3
linter-xmllint, v0.0.7
python-jedi, v0.3.4
rst-preview, v1.3.0
rst-preview-pandoc, v0.1.11
set-syntax, v0.3.0
seti-ui, v0.8.1
tabs-to-spaces, v0.11.1
visual-studio-dark-syntax, v0.1.0

# Dev
No dev packages
```

**Comment by Krzysztof-Cieslak** [comment]
Introduced in latest version?

CC: @jeroldhaas

**Comment by stweb** [comment]
same here on OSX 10.11.1 after today's update

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `1.0.3`

**Comment by stweb** [comment]
Thanks for the quick turnaround and all your efforts on this project!

**Comment by jeroldhaas** [comment]
I thought I'd added that semi-colon, but it appears I'd missed it or maybe changed it after I'd committed.

Sorry for this, and ref #5 ...

**Comment by Yoshi325** [comment]
Just to roundtrip it: It's fixed for me by 1.0.3!
